TombEngine/Scripting/Internal/TEN/Misc/LogLevel.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <unordered_map>
+#include <string>
+
+/***
+Constants for LogLevel IDs.
```

Comment:
It would be good to put example usage like WeaponType does.

Reply:
I added an example with possible values in the function description